website/source/docs/jobops/inspecting.html.md

```diff
@@ -0,0 +1,169 @@
+Status             = complete
+Status Description = complete
+Type               = service
+TriggeredBy        = job-register
+Job ID             = example
+Priority           = 50
+Placement Failures = true
+
+Failed Placements
+Task Group "cache" (failed to place 3 allocations):
+  * Resources exhausted on 1 nodes
+  * Dimension "cpu exhausted" exhausted on 1 nodes
+
+Evaluation "5744eb15" waiting for additional capacity to place remainder
+```
+
+More interesting though is the [`alloc-status`
+command](/docs/commands/alloc-status.html). This command gives us the most
+recent events that occured for a task, its resource usage, port allocations and
+more:
+
+```
+nomad alloc-status 12
+ID            = 12681940
+Eval ID       = 8e38e6cf
+Name          = example.cache[1]
+Node ID       = 4beef22f
+Job ID        = example
+Client Status = running
+
+Task "redis" is "running"
+Task Resources
+CPU    Memory           Disk     IOPS  Addresses
+2/500  6.3 MiB/256 MiB  300 MiB  0     db: 127.0.0.1:57161
+
+Recent Events:
+Time                   Type        Description
+06/28/16 15:46:42 UTC  Started     Task started by client
+06/28/16 15:46:10 UTC  Restarting  Task restarting in 30.863215327s
+06/28/16 15:46:10 UTC  Terminated  Exit Code: 137, Exit Message: "Docker container exited with non-zero exit code: 137"
+06/28/16 15:37:46 UTC  Started     Task started by client
+06/28/16 15:37:44 UTC  Received    Task received by client
+```
+
+In the above example we forced killed the docker container so that we could see
+in the event history that Nomad detected the failure and restarted the
+allocation.
+
+The `alloc-status` command is a good starting to point for debugging an
+application that did not start. In this example task we are trying to start a
+redis image using `redis:2.8` but the user has accidentally put a comma instead
+of a period, typing `redis:2,8`.
+
+
+When the job is run, it produces an allocation that fails. The `alloc-status`
+command gives us the reason why:
+
+```
+nomad alloc-status c0f1
+ID            = c0f1b34c
+Eval ID       = 4df393cb
+Name          = example.cache[0]
+Node ID       = 13063955
+Job ID        = example
+Client Status = failed
+
+Task "redis" is "dead"
+Task Resources
+CPU  Memory   Disk     IOPS  Addresses
+500  256 MiB  300 MiB  0     db: 127.0.0.1:23285
+
+Recent Events:
+Time                   Type            Description
+06/28/16 15:50:22 UTC  Not Restarting  Error was unrecoverable
+06/28/16 15:50:22 UTC  Driver Failure  failed to create image: Failed to pull `redis:2,8`: API error (500): invalid tag format
+06/28/16 15:50:22 UTC  Received        Task received by client
+```
+
+Not all failures are this easily debuggable. If the `alloc-status` command shows
+many restarts occuring as in the example below, it is a good hint that the error
+is occuring at the application level during start up. These failres can be
+debugged by looking at logs which is covered [here](/docs/jobops/logs.html).
```

Comment:
style nit: never use "here" as a hyperlink.  "Debugged by looking at the logs covered in the [Nomad Job Logging Documentation](...)."